src/main/kotlin/no/nav/tilleggsstonader/sak/utbetaling/tilkjentytelse/domain/TilkjentYtelse.kt

```diff
@@ -26,7 +27,8 @@ data class TilkjentYtelse(
     @Id
     val id: UUID = UUID.randomUUID(),
     val behandlingId: UUID,
-    val andelerTilkjentYtelse: List<AndelTilkjentYtelse>,
+    @MappedCollection(idColumn = "tilkjent_ytelse_id")
+    val andelerTilkjentYtelse: Set<AndelTilkjentYtelse>,
```

Comment:
Hvorfor er det set og ikke liste? For unike verdier?

Reply:
List stötte ikke av data jdbc